shell/src/main/java/org/apache/accumulo/shell/commands/GrepCommand.java

```diff
@@ -71,10 +71,11 @@ public int execute(final String fullCommand, final CommandLine cl, final Shell s
       scanner.setTimeout(getTimeout(cl), TimeUnit.MILLISECONDS);
 
       setupSampling(tableName, cl, shellState, scanner);
+      addScanIterators(shellState, cl, scanner, "");
```

Comment:
Does this change also benefit the EgrepCommand?

Reply:
Yes, ergrep uses grep execute() method